src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/WorkingDayCountService.kt

```diff
@@ -43,3 +40,18 @@ class WorkingDayCountService(
     return date.getNextWorkingDay(bankHolidays)
   }
 }
+
+fun interface BankHolidaysProvider {
+  fun getUKBankHolidays(): List<LocalDate>
+}
+
+@Service
+class GovUkBankHolidaysProvider(
```

Comment:
do we need to add a unit test for the new `GovUkBankHolidaysProvider` to describe the success and failure responses? Does that come later?